2020/cmd/08/vm.go

```diff
@@ -22,6 +22,7 @@ type accInstr struct {
 	v int
 }
 
+// QUESTION: is `*accInstr` ok, or I would be better of with `accInstr`? What's the difference?
```

Comment:
When using value receivers you're working on a copy of a receiver. For such small structs using value receivers is recommended unless you want to mutate the receiver. 
So `accInstr` is better here.